dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/communityManager/DataRequestUploadListenerTest.kt

```diff
@@ -116,15 +118,15 @@ class DataRequestUploadListenerTest {
     fun `patch your own data request as a premium user to closed and check that this is allowed if answered before`() {
```

Comment:
more readable if ´the text would be: "assert that patching ... is forbidden"

Reply:
changed the name, although I didnt created this function